packages/create/templates/hello-express/dbos-config.yaml

```diff
@@ -0,0 +1,20 @@
+# To enable auto-completion and validation for this file in VSCode, install the RedHat YAML extension
+# https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml
+
+# yaml-language-server: $schema=https://raw.githubusercontent.com/dbos-inc/dbos-transact/main/dbos-config.schema.json
+
+language: node
+database:
+  hostname: localhost
+  port: 5432
+  username: postgres
+  password: ${PGPASSWORD}
+  connectionTimeoutMillis: 3000
+  app_db_client: knex
+  migrate:
+    - npx knex migrate:latest
+  rollback:
+    - npx knex migrate:rollback
+runtimeConfig:
+  entrypoints:
+    - dist/main.js
```

Comment:
This is the part I don't understand. Since the app is built with Express and doesn't use our HTTP decorators, shouldn't it use the new start command instead of an entrypoint?

Reply:
Start command did not exist when that line was written.

Reply:
The entrypoints are used to inform Transact where to load classes for registrations

Reply:
You may have noticed that I cheated and took advantage of the fact that the entrypoint file is essentially "run".  This can be improved with the new facilities.

Reply:
Now with the start command, should we change this?